tests/conftest.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Optional
+
+import pytest
+
+from mkosi.config import OutputFormat
+from mkosi.distributions import Distribution, detect_distribution
+
+
+def pytest_addoption(parser: Any) -> None:
+    def parse_integration_distribution(s: str) -> set[Optional[Distribution]]:
+        if s == "hostonly":
+            return {detect_distribution()[0]}
+        return {Distribution(d) for d in s.split(",")}
+
+    def parse_integration_output(s: str) -> set[OutputFormat]:
+        if s == "all":
+            return {f for f in OutputFormat if f != OutputFormat.none}
+        return {OutputFormat(o) for o in s.split(",")}
+
+    parser.addoption(
+        "-D",
+        "--integration-distribution",
+        metavar="DISTRO",
+        help="Run the integration tests for the given DISTRO.",
+        default="hostonly",
+        type=parse_integration_distribution,
+    )
+    parser.addoption(
+        "-O",
+        "--integration-output",
+        metavar="OUTPUT",
+        help="Run the integration tests for the given OUTPUT.",
+        default="all",
+        type=parse_integration_output,
+    )
```

Comment:
I would still drop this, you can already do this via the test name "test_boot[disk]". Until we add more tests parametrized over the output format we shouldn't add this.

Reply:
Dropped it, although even so I found it much more convenient then typing out the test instance name